platform/admin/core/admin-core-appservice/src/main/java/org/codice/ddf/admin/application/service/util/JsonUtils.java

```diff
@@ -11,7 +11,7 @@
  * License is distributed along with this program and can be found at
  * <http://www.gnu.org/licenses/lgpl.html>.
  */
-package org.codice.ddf.admin.application.service.migratable;
+package org.codice.ddf.admin.application.service.util;
```

Comment:
❔ Are we keeping this because JsonUtils have a general purpose to it outside migratable?

Reply:
Yup, there were uses of this module from other modules even though it was in migratable.